app/liquid/liquid_renderer.rb

```diff
@@ -134,6 +134,7 @@ def isolate_from_plugin_data(field)
 
   def location
     return @location if @location.blank?
+    return {} if @location.data['country_code'] == 'RD'
```

Comment:
Do you know what fields are we actually using from here? I could just return a smaller object with the data that we need.